com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -397,11 +414,22 @@ internal Vector3 Scale
         /// <param name="dirtyTime"></param>
         protected void TryCommitTransformToServer(Transform transformToCommit, double dirtyTime)
         {
+            TryCommitTransform(transformToCommit, dirtyTime);
+        }
+
+
+        protected void TryCommitTransform(Transform transformToCommit, double dirtyTime)
```

Comment:
Probably worth moving changing the `TryCommitTransformToServer` summary to something like `Prefer <see cref="TryCommitTransform">. Legacy method.`